db/config.js

```diff
@@ -4,7 +4,66 @@ mongoose.connect('mongodb://localhost/test', {useNewUrlParser: true});
 
 const db = mongoose.connection;
 
-db.on('error', console.error.bind(console, 'connection error:'));
-db.once('open', function() {
-  // we're connected!
-});
\ No newline at end of file
+// db.collection("test").findOne({}, function(err, result) {
+//   if (err) throw err;
+//   console.log(result);
+//   db.close();
+// });
+
+
+
+
+
+const getAll = (callback) => {
+  db.collection("test").find({}).toArray((err, result) => {
+    if (err) {
+      callback(err)
+    } else {
+      callback(null, result)
+    }
+  })
+}
+
+
+
+
+// MongoClient.connect(url, function(err, db) {
+//   if (err) throw err;
+//   var dbo = db.db("mydb");
+//   dbo.collection("customers").find({}).toArray(function(err, result) {
+//     if (err) throw err;
+//     console.log(result);
+//     db.close();
+//   });
+// });
+
+
+module.exports = {
+  getAll : getAll,
```

Comment:
A common pattern I've been seeing is the[ property shorthand functionality in ES6](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Property_definitions). You can just list the property name if your key/value pair are the same.
`module.exports = { getAll };`

Reply:
updated